monitor/alerts/test.py

```diff
@@ -15,11 +14,21 @@
 logger = get_logger(__name__)
 
 
-@dataclass
 class TestAlert(Alert):
-    model_unique_id: str
-    test_unique_id: str
-    status: str
+    def __init__(
```

Comment:
Convert from dataclass into class.
1. IMO dataclass should be used in very specific situations when the class is proper dataclass (and usually it will be better to use pydantic schema)
2. dataclass and inheritance doesn't work so well, and we inheritance (with default values) this class.